Lib/fontmake/font_project.py

```diff
@@ -172,6 +180,9 @@ def build_master_ufos(
         except Exception as e:
             raise FontmakeError("Loading Glyphs file failed", glyphs_path) from e
 
+        if font.customParameters["Enforce Compatibility Check"]:
```

Comment:
traditionally, when there are Glyphs-only custom parameters involved, given that the pipeline works with UFOs internally, we have looked those up inside the UFO or Designspace libs (as exported by glyphsLib from the original .glyphs source) instead of directly in the GSFont instance. This makes it possible for DS+UFO workflows to avail themselves of these, or to build from .glyphs source in stages (first export to DS+UFO, then maybe do something else, then continue on to build off that).

If you check the GSFont custom parameter directly, it's not possible for a DS worfklow to store this option anywhere, requiring one to pass the CLI option.

Ideally we'd have our own (com.github.googlefonts.fontmake..) lib key that stores that in the DS.lib and glyphsLib writes that out. Maybe we can add later as needed

Reply:
Oh man. This is like playing Jenga. :-)

Reply:
welcome to my life 😅

Reply:
I think you also need to add `check_compatibility` to `exclude_args` in here, just before running pipeline from single UFOs:

https://github.com/googlefonts/fontmake/blob/f36543baf5065f05e7fb0c5ea58d7eab8bc9255a/Lib/fontmake/__main__.py#L572-L584